iot_hub.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
As above, shebang and `__main__` are recommended

Reply:
Sounds good, updating these.